config/v1/types_ingress.go

```diff
@@ -90,8 +90,28 @@ type IngressSpec struct {
 	// provider of the current cluster and are required for Ingress Controller to work on OpenShift.
 	// +optional
 	LoadBalancer LoadBalancer `json:"loadbalancer,omitempty"`
+
+	// logLevel describes the desired logging verbosity for the Ingress Operator
+	// Any one of the following values may be specified:
+	// * Normal
+	// * Debug
+	// Defaults to "Normal".
+	// +optional
+	// +kubebuilder:default=Normal
+	OperatorLogLevel IngressOperatorLogLevel `json:"operatorLogLevel,omitempty"`
```

Comment:
Why is it defined in this file ? This file is for config ingress object. Are you going to define log levels in ingress config object ? or in Ingress Controller CR ?

Reply:
Operator log level doesn't need to be defined for each ingress controller, since there's only one instance of the operator for any number of instances of ingress controller resource. I think the ingress config object is the only logical place for it.

Reply:
How would that work if you have mutliple ingress controllers, with each specifying `operatorLogLevel`?

Reply:
Each has its own pod, its own specification, and its own log.  If I have one shard/controller for prod and one for dev, and I want to log them differently, I would specify the operalorLogLevel on each one's spec, right?

Reply:
There's only one operator instance per cluster, not per ingress controller

Reply:
I have a hunch that if you remove this from config/v1 it might solve your integration test problem.

Reply:
As others have commented, this isn't the spot you're looking for.  This gets defined one level lower on the operator.openshift.io.  The types in config.openshift.io are logical units, but aren't intended to correspond to individual operators.

Reply:
I'm not sure where else it could go, though, because the only ingress-related API resource in operator.openshift.io is `ingresscontroller`, and that's not for configuration of the operator either. 

Do we need an additional custom resource for operator config? That feels like overkill for just `operatorLogLevel`, but maybe there are other existing/new APIs that make sense there.